doc/COOKBOOK-FIXERS.md

```diff
@@ -128,7 +128,7 @@ final class RemoveCommentsFixerTest extends AbstractFixerTestCase
 
     public function provideFixCases()
     {
-        return array();
+        return [];
```

Comment:
I'm not asking you to change branch, but dropping short array syntax changes (and then, if you want, open new PR for 2.3 line changing them)